plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergSystemTables.java

```diff
@@ -683,4 +684,24 @@ private BaseTable loadTable(String tableName)
     {
         return IcebergTestUtils.loadTable(tableName, metastore, fileSystemFactory, "hive", "tpch");
     }
+
+    @Test
```

Comment:
BaseIcebergSystemTables is for testing metadata tables. Please move to `BaseIcebergConnectorTest`.

Reply:
shall we leave the test here after added $partitions select?

Reply:
The test purpose is verifying CTAS success, right? BaseIcebergConnectorTest is more suitable if so.